src/main/java/utils/StackUtil.java

```diff
@@ -0,0 +1,121 @@
+package utils;
+
+import linkedlist.Node;
+import linkedlist.SingleLinkedList;
+
+import java.util.Stack;
+
+public class StackUtil {
```

Comment:
All function should be write down the requirement or example to make sure we know the purpose your implement.